src/handlers/transformation/transform-csv-to-json.ts

```diff
@@ -0,0 +1,15 @@
+import { S3Event } from "aws-lambda";
+import * as AWS from "aws-sdk";
+import csv from "csvtojson";
+
+// Unsure how to test this function or indeed if it even needs testing given that it basically just using S3 and the csvtojson library. Feedback/Help welcome!
+export async function transformCsvToJson(event: S3Event): Promise<any[]> {
```

Comment:
I do believe this function could maybe live in shared/utils/S3.ts though, next to readJsonFromS3, as it's doing a very similar thing.  Maybe rename it to something like readJsonFromS3Csv to match the function names there.